### PR TITLE
Add all analysis page and claim details

### LINKED ISSUE #234: All analysis screen
as part of the dashboard refactor, we need to implement the "All analysis" screen. Its a historical page that allows the user to see all his/her past votes on claims. Its scoped by community and contains filters. It also allows users to review pending claims.

Figma designs: https://www.figma.com/file/nEPG6Dkns8hDVCudVUTEzc/Dashboard?node-id=572%3A505

### PR DESCRIPTION
**Pull request by lucca65**
## What issue does this PR close
Closes #234 

## Changes Proposed ( a list of new changes introduced by this PR)
- Add "All actions for analysis" screen. Here the user can see all the analysis, the ones he still have to do, the ones he did, and the status of the claims. 

## How to test ( a list of instructions on how to test this PR)
You'll have to be a validator to at least one valid claim on any community you belong to.

- Open the dashboard, if you are a validator in at least one valid claim you'll be able to see the "Actions for analysis" panel. If not, check your selected community on the header
- Click on "All"
- See all the history
- You can check the details for a older vote on a claim by clicking "More details"
- On the more details screen you should see the voting status

### CODE REVIEW

**Comment by lucca65** [comment]
Folks there are a few warnings on unused variables, that is because I had to breakdown this PR into 3 parts:

1) New all analysis page with pagination and claim status page
2) New analysis filters (#240)
3) Add pagination to dashboard (#241)

Those unused variables will be used the PR that closes those two other issues

File: src/elm/Page/Dashboard/Analysis.elm [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+
+            else if Claim.isAlreadyValidated claim accountName then
+                ( t "dashboard.all_analysis.pending", "text-black" )
+
+            else
+                ( t "dashboard.all_analysis.disapproved", "text-red" )
+    in
+    div [ class "w-full sm:w-full md:w-1/2 lg:w-1/3 xl:w-1/4 px-2 mb-4" ]
+        [ if Claim.isAlreadyValidated claim accountName then
+            div [ class " flex flex-col p-4 my-2 rounded-lg bg-white" ]
```
Comment (amiskov) [inline src/elm/Page/Dashboard/Analysis.elm:182]: Redundant space at the beginning of a class string.

File: src/elm/Page/Dashboard/Analysis.elm [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+                    ]
+                , a
+                    [ class "button button-secondary w-full font-medium mb-2"
+                    , Route.href <| Route.Claim selectedCommunity claim.action.objective.id claim.action.id claim.id
+                    ]
+                    [ text_ "dashboard.all_analysis.more_details" ]
+                ]
+
+          else
+            div [ class " flex flex-col p-4 my-2 rounded-lg bg-white", id <| "claim-" ++ String.fromInt claim.id ]
```
Comment (amiskov) [inline src/elm/Page/Dashboard/Analysis.elm:207]: One more redundant space :)

**Comment by amiskov** [comment]
Regarding the conflict with master-branch in `src/styles/main.css`: I replaced `h-10` with `h-12` after overlaying Helton's mock-ups with the Pixelperfect plugin. `h-12` had turned out more suitable for buttons height.

**Comment by lucca65** [comment]
@heltonlr can you please test it? Developers already approved. Its on app.cambiatus.io

**Pull request merged**